layers/+completion/auto-insert-template/packages.el

```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- auto-insert-template layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Yasuharu Iida <YasuharuIida@users.noreply.github.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst auto-insert-template-packages
+  '(
+    (autoinsert :location built-in)
+    yasnippet
```

Comment:
If there's no function associated with these packages in your packages.el file, they shouldn't be listed in the package list either.